cpp/examples/graph.cpp

```diff
@@ -31,20 +31,20 @@ int main()
     const auto tmax = 10.;
     const auto dt   = 0.5; //time step of migration, daily migration every second step
 
-    mio::oseir::Model model;
-    model.populations[{mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Susceptible)}] = 10000;
+    mio::oseir::Model model(1);
+    model.populations[{mio::Index<mio::AgeGroup>(0),mio::Index<mio::oseir::InfectionState>(mio::oseir::InfectionState::Susceptible)}] = 10000;
     model.parameters.set<mio::oseir::TimeExposed>(1);
-    model.parameters.get<mio::oseir::ContactPatterns>().get_baseline()(0, 0) = 2.7;
+    model.parameters.get<mio::oseir::ContactPatterns>().get_cont_freq_mat()[0].get_baseline().setConstant(2.7);
```

Comment:
i think using 
```
mio::ContactMatrixGroup& contact_matrix = params.get<mio::oseir::ContactPatterns>();
    contact_matrix[0] =....
```
would be more useful here. And easier to change for potential age groups